artistic/templates/application.html

```diff
@@ -22,6 +22,17 @@
         {% block content %}{% endblock %}
       </div>
     </div>
+    <div class="col-md-8">
+      {% with messages = get_flashed_messages(with_categories=true) %}
+        {% if messages %}
+          {% for category, message in messages %}
+            <div class="alert alert-{{ category }}" style="text-align: center;">
+              {{ message }}
+            </div>
+          {% endfor %}
+        {% endif %}
+      {% endwith %}
+    </div>
```

Comment:
This would show alerts at the bottom of our form, right? Usually they appear at the top. Could you shift it below the nav and above the content?

Reply:
Moved alert between nav and content.